e2e/cypress/pages/newCasetype.page.js

```diff
@@ -7,36 +7,34 @@ const TEMPLATE = '[data-cy=template]';
 const SAVE_BUTTON = '[data-cy=save-btn]';
 
 export default class NewCaseTypePage {
-
-  visitPage = () => {
-    cy.log('navigating to %s', Urls.NEW_CASETYPE_URL);
-    cy.visit(Urls.NEW_CASETYPE_URL);
-    return this;
-  };
-
-  typeName = (value) => {
-    cy.get(NAME).type(value);
-    return this;
-  };
-
-  selectPartyType = (value) => {
-    cy.get(PARTY_TYPE).select(value);
-    return this;
-  };
-
-  selectTeam = (value) => {
-    cy.get(TEAM).select(value);
-    return this;
-  };
-
-  typeTemplate = (value) => {
-    cy.get(TEMPLATE).type(value);
-    return this;
-  };
-
-  save = () => {
-    cy.get(SAVE_BUTTON).click();
-    return this;
-  };
-
+    visitPage = () => {
+        cy.log('navigating to %s', Urls.NEW_CASETYPE_URL);
+        cy.visit(Urls.NEW_CASETYPE_URL);
+        return this;
+    };
+
+    typeName = (value) => {
+        cy.get(NAME).type(value);
+        return this;
+    };
+
+    selectPartyType = (value) => {
+        cy.get(PARTY_TYPE).select(value);
+        return this;
+    };
+
+    selectTeam = (value) => {
+        cy.get(TEAM).select(value);
+        return this;
+    };
+
+    typeTemplate = (value) => {
+        cy.get(TEMPLATE).type(value);
```

Comment:
can you change this so that cypress injects the whole text directly? Currently, the text is being inputed character by character, which is quite long.

Perhaps in your page object you could have something like

```suggestion
        cy.get(TEMPLATE).value = value
```